src/Unleash.php

```diff
@@ -14,4 +14,6 @@ public function isEnabled(string $featureName, ?Context $context = null, bool $d
     public function getVariant(string $featureName, ?Context $context = null, ?Variant $fallbackVariant = null): Variant;
 
     public function register(): bool;
+
+    public function refreshCache(): void;
```

Comment:
This IMO doesn't really belong on the Unleash interface, it should be called as part of the repository.

Reply:
I don't like that either, do you have an idea how to access the functionality then? I only did that to expose it to users.

Reply:
Currently you would need to provide your own repository to have access to that, so that's an option. Other than that, rework of the builder might be due so that it can return not only the `Unleash` object, but also the intermediate objects.

@sighphyre What's your opinion?

Reply:
> This IMO doesn't really belong on the Unleash interface, it should be called as part of the repository.

Very much agree. I don't think it belongs in the top level API, and I think the `BadMethodCallException` in the `DefaultProxyUnleash` is a warning sign that the abstraction is subtly off. 

I can't really see forced cache updates being called from the same abstraction layers as `isEnabled` /`getVariant` in user code (entirely open to the idea that I just don't understand how PHP apps are architected though). Forced cache updates feels like a thing I want called in another thread/process, to me that says that right now the `Repository` is the correct layer for that to live and that maybe this PR is currently a stepping stone to having an updater/scheduler component

Reply:
@michael-schmetter Please remove the method from the `Unleash` interface altogether, the PR is fine without it, solving how to get the Repository object out of the builder should be a separate PR.

Currently the only way to have access to repository is by manually constructing the whole object (as seen [in the example](https://github.com/Unleash/unleash-client-php/blob/main/examples/07-manual-unleash-object.php)).

In a separate PR we can do one of two things:

- make it possible to set a custom repository on the builder
- rework the builder to return intermediate objects as well

If you'd like to tackle it, let me know, otherwise I can do the PR myself.

Reply:
@RikudouSage 
I updated the PR and removed all superfluous changes that were related to the `Unleash` interface.

I can do a PR, but let's quickly connect, what you would have envisioned.

something like 

```
list($unleash, $repository) = UnleashBuilder::create()
    ->withInstanceId('Some instance id')
    ->buildWithRepository();
```

Or would you go another route?

Reply:
I'd go in the direction of there being multiple methods, like `buildRepository()` and `build()` and possibly others. The `build()` would internally reuse the `buildRepository()` etc.